packages/mdc-list/README.md

```diff
@@ -364,6 +364,7 @@ Method Signature | Description
 `setSingleSelection(value: Boolean) => void` | Sets the list to be a selection list. Enables the `enter` and `space` keys for selecting/deselecting a list item.
 `setSelectedIndex(index: Number) => void` | Toggles the `selected` state of the list item at index `index`.
 `setUseActivated(useActivated: boolean) => void` | Sets the selection logic to apply/remove the `mdc-list-item--activated` class.
+`setFocusInListItemChildren(enable: Boolean) => void` | Sets the list to enable or disable focusing in list item children.
```

Comment:
I think this name doesn't indicate what it's enabling or disabling. Maybe `enableToggleTabindexForChildren` ? That's probably worse. We should at least add more to the description to indicate what it's doing.  Maybe try to link this description back to the description on 271-273.

Reply:
Done.

Reply:
How about...

```
`setTabbableChildren(tabbable: boolean) => void` | Sets whether focusable children of the currently-focused list item can also be individually focused via the Tab key.
```